d2game/d2player/game_controls.go

```diff
@@ -28,7 +28,6 @@ const (
 // Panel represents the panel at the bottom of the game screen
 type Panel interface {
 	IsOpen() bool
-	Toggle()
```

Comment:
what do you think, if I'm removing it, should I remove Toggle() function from all panels' widgets?

Reply:
I vote yes